profiles/models.py

```diff
@@ -1,3 +1,16 @@
 from django.db import models
 
-# Create your models here.
+from users.models import Users
+
+
+class StartUpProfile(models.Model):
+    user_id = models.ForeignKey(Users, on_delete=models.CASCADE)  # model User is empty
+    name = models.CharField(unique=True, max_length=255, blank=False)
+    description = models.TextField()
```

Comment:
You might want to add a constraint to enforce a maximum length for descriptions:
```
description = models.TextField(max_length=1000)
```